samples/drivers/espi/prj_mec15xxevb_assy6853.conf

```diff
@@ -0,0 +1,13 @@
+# eSPI + mec15xxevb_assy6853
```

Comment:
what is the point of this file? we already have a board specific .conf file already.

Reply:
@nashif 
prj.conf is for any non-mchp board that may support eSPI in the future. 
mec15xxevb_assy6853 configuration is main validation vehicle board.
mec15modular is very limited in standalone mode for validation, requires to be used in conjunction with Intel RVP
We need separate files to enable different features in each board